app/authentication/token.py

```diff
@@ -45,6 +47,25 @@ async def is_admin(token: str = Depends(oauth2_scheme)) -> bool:
     else:
         return True
 
+# Check is the authorized user is an admin. Return true if so, false if not (with no
+# exception).
+async def is_admin_no_exception(token: str = Depends(oauth2_scheme)) -> bool:
+    """Calls GFW API to authorize user.
+
+    User must be ADMIN for gfw app
+    """
+
+    response: Response = await who_am_i(token)
+
+    if response.status_code == 401 or not (
+        response.json()["role"] == "ADMIN"
+        and "gfw" in response.json()["extraUserData"]["apps"]
```

Comment:
if this dataset is to be used just by GFW Pro, would could consider further narrowing down access to users with `gfw-pro` in their `apps` object.

Reply:
Yes, I changed for checking for the gfw-pro admin in the case of the licensed WDPA dataset.  Thanks for the suggestions!